docs/en/how-to-prepare-for-the-transfer-gateway-update.md

```diff
@@ -0,0 +1,113 @@
+
+The way in which you must instantiate the Ethereum Transfer Gateway used for withdrawals has changed. That said, you need to import the `createEthereumGatewayAsync` function from `loom-js` using something like this:
+
+```js
+import {
+  LocalAddress,
+  CryptoUtils,
+  Address,
+  Contracts,
+  createEthereumGatewayAsync, //Include this line!
+  getMetamaskSigner
+} from 'loom-js'
+```
+
+Then, you can instantiate the gateway as follows:
+
+```js
+async _getEthereumTransferGatewayContract(web3Ethereum) {
+  const networkId = await web3Ethereum.eth.net.getId()
+  console.log("networkId", networkId);
+
+  let version
+  switch (networkId) {
+    case 1: // Ethereum Mainnet
+      version = 1
+      break
+
+    case 4: // Rinkeby
+      version = 2
+      break
+    default:
+      throw new Error('Ethereum Gateway is not deployed on network ' + networkId)
+  }
+
+  const signer = getMetamaskSigner(web3Ethereum.currentProvider) // If you're running `loom-js` in Node.js, change this line to something like `const signer = new OfflineWeb3Signer(rinkeby.web3js, rinkeby.account)
+
+  this.ethereumGatewayContract = await createEthereumGatewayAsync(
+    version,
+    this._RinkebyGatewayAddress(), // In this example, we're instantiating the Rinkeby transfer gateway
+    signer
+  )
+}
+```
+
+### 3.2 Use the withdrawalReceiptAsync Function to Get the Withdrawal Receipt
+
+Prior to `loom-js` 1.70.1, you've probably used something similar to the following to get the withdrawal receipt:
+
+```js
+const data = await gatewayContract.withdrawalReceiptAsync(userLocalAddr)
+if (!data) {
+  return null
+}
+const signature = CryptoUtils.bytesToHexAddr(data.oracleSignature)
+return {
+  signature: signature,
+  amount: data.value.toString(10),
+  tokenContract: data.tokenContract.local.toString()
+}
+```
+
+With `loom-js` 1.70.1, you can replace this snippet with an one-liner:
+
+```js
+const receipt = await gatewayContract.withdrawalReceiptAsync(userLocalAddr)
+```
+
+### 3.3 Call the withdrawAsync Function to Withdraw Your Assets from the Ethereum Gateway
+
+We've also streamlined the way in which assets get withdrawn from the transfer gateway. Instead of calling different methods depending on what asset you want to withdraw, with 1.70.1, you can use the following snippet:
+
+```js
+const tx = await gatewayContract.withdrawAsync(receipt, { gasLimit: gas })
+return tx.hash
+```
+
+Please note that you must also call the `withdrawAsync` function to resume a withdrawal.
+
+## Sample Code
+
+We've updated our examples to reflect the changes to the API. If you're running `loom-js` in Node.js, please refer to the [truffle-dappchain-example](https://github.com/loomnetwork/truffle-dappchain-example/) repository for more details. If you're running `loom-js` in the browser, please see the [loom-examples](https://github.com/loomnetwork/loom-examples) repository.
```

Comment:
Should link to the gateway-cli.js file in the truffle dappchain example, instead of just the repo root.